src/utils/utils.ts

```diff
@@ -31,3 +32,15 @@ export async function executeCommand(command: string, cwd?: string): Promise<str
 export function generateQuotedFullName(schema: string, objectName: string): string {
 	return `[${escapeClosingBrackets(schema)}].[${escapeClosingBrackets(objectName)}]`;
 }
+
+/**
+ * Promise with timeout to check for another promise fulfillment
+ * @returns a promise that resolves after the specified timeout
```

Comment:
This isn't true - it never resolves. It _rejects_ after the specified timeout.

And you should either say here that the timeout is 10sec or have the timeout be a parameter (with a default of 10sec)